lib/nfs_common.pm

```diff
@@ -247,8 +247,14 @@ sub start_service {
     # Back on the console
     clear_console;
 
+    # Duo to poo#114875, the below journalctl command may take several time to
+    # get return code in serial terminal on low performance setups. so enhance
+    # the logic not to check the return code and add some sleep here to make
+    # the test more stable.
+    enter_cmd "( journalctl -fu nfs-server -o short-precise > /dev/$serialdev & )";
+    sleep 3;
```

Comment:
You can increase the timeout if you want to still get the journal log, and not have it failing due to slow performance, but let's not add a `sleep` unless it's absolutely necessary

```suggestion
    enter_cmd "( journalctl -fu nfs-server -o short-precise > /dev/$serialdev )";
```

Reply:
Thanks @foursixnine, As per my test result `sleep 3` on aarch64 platform should be added here, since it took some time to flush the output to serial terminal. 

As I mentioned in the ticket 114854, Even I increased the script_run timeout to 90s, I can't fix the issue 

`( journalctl -fu nfs-server -o short-precise > /dev/ttyAMA0 & ); echo qBERp-$?-
zypper -n in mariadb | cat; ( exit ${PIPESTATUS[0]} ); echo 7UoWa-$?-# ( journalctl -fu nfs-server -o short-precise > /dev/ttyAMA0 & ); echo qBERp-$?-

qBERp-0-

I can suppose that serial is busy there and ( journalctl -fu nfs-server -o short-precise > /dev/ttyAMA0 & ) may take some time to get return.

Let me try to use "enter_cmd" instead of "script_run"`

Later, `enter_cmd` can get better result.  however without `sleep 3`, similar issue can still be seen.  

Let me try to add `sleep 3` on aarch64 platform only and see

Reply:
Thanks @foursixnine, As per my test result `sleep 3` on aarch64 platform should be added here, since it took some time to flush the output to serial terminal. 

As I mentioned in the ticket 114854, Even I increased the script_run timeout to 90s, I can't fix the issue 

`( journalctl -fu nfs-server -o short-precise > /dev/ttyAMA0 & ); echo qBERp-$?-
zypper -n in mariadb | cat; ( exit ${PIPESTATUS[0]} ); echo 7UoWa-$?-# ( journalctl -fu nfs-server -o short-precise > /dev/ttyAMA0 & ); echo qBERp-$?-

qBERp-0-

I can suppose that serial is busy there and ( journalctl -fu nfs-server -o short-precise > /dev/ttyAMA0 & ) may take some time to get return.

Let me try to use "enter_cmd" instead of "script_run"`

Later, `enter_cmd` can get better result.  however without `sleep 3`, similar issue can still be seen.  

Let me try to add `sleep 3` on aarch64 platform only and see

Reply:
@foursixnine http://openqa.suse.de/tests/overview?distri=sle&version=15-SP4&build=rfan_nfs_server_entercmd&test=qam-minima_%2Bbase_RFAN
Test passed on all platforms.

BTW, `enter_cmd "( journalctl -fu nfs-server -o short-precise > /dev/$serialdev )";` seems not work since it can't get return code if we don't run it in for-ground.